docs/resources/performance-guidance.md

```diff
@@ -0,0 +1,62 @@
+
+## Azure SQL Database Tier
+
+The Standard tier of the DTU-based SQL performance tiers is enabled by default upon deployment. We recommend the DTU Purchase Model over the vCore model for the Medical Imaging Server for DICOM. In DTU-based SQL purchase models, a fixed set of resources is assigned to the database via performance tiers: Basic, Standard and Premium.
+
+To review the various SQL Database Tiers from Azure, refer to [Azure SQL Database Pricing](https://azure.microsoft.com/pricing/details/sql-database/single/). To learn how to update your Azure SQL Database tier, refer to [Configure Medical Imaging Server for DICOM Settings](../how-to-guides/configure-dicom-server-settings.md).
+
+## Geo-Redundancy
+
+For a production workload, we highly recommend configuring your Medical Imaging Server for DICOM to support geo-redundancy.
+
+### Geo-Redundant Azure Storage
+
+Azure Storage offers geo-redundant storage to ensure high availability even in the event of a regional outage. We highly recommend choosing an Azure Storage Account Sku that supports geo-redundancy if you are running a production workload. Azure storage offers two options for geo-redundant replication, Geo-zone-redundant-storage (GZRS) and Geo-redundant-storage (GRS). Refer to this article to decide which geo-redundant Azure Storage option is right for you: [Use geo-redundancy to design highly available applications](https://docs.microsoft.com/en-us/azure/storage/common/geo-redundant-design).
+
+You can customize your Azure Storage Account SKU during deployment via the Medical Imaging Server for DICOM [Quickstart Deploy to Azure](../quickstarts/deploy-via-azure.md). By default, Standard LRS is selected, which is Standard Locally Redundant Storage.
+
+### Geo-replication for SQL Database
+
+In addition to configuring geo-redudunant Azure Storage, we recommend configuring active geo-replication for your Azure SQL Database. This allows you to create readable secondary databases of individual databases on a server in the same or different data center region. For a tutorial on how to configure this, see [Creating and using active geo-replication - Azure SQL Database](https://docs.microsoft.com/azure/azure-sql/database/active-geo-replication-overview).
+
+## Workload Scenarios for Azure SQL Database & Azure App Service
+
+### Scenario 1: Testing out DICOM Server
+
+If you are testing out the Medical Imaging Server for DICOM and not running production workloads, we recommend using the S1 Azure App Service Tier alongside a Standard Azure SQL Database (S1, S2, S3). For a small system that does not require redundancy, with these tiers, you can spend as little as ~$70/month on Azure App Service & Azure SQL Database.
+
+If you had ~100,000 DICOM instances, you should expect a range of 2000-9000 requests/minute and a response time under 1 second for STOW, WADO & QIDO.
```

Comment:
I don't think referring to the number of instances here makes sense! The numbers should hold decently for higher numbers as well. The key would be to monitor the percentage of SQL DB usage.